systemtest/src/test/java/io/strimzi/systemtest/operators/user/UserST.java

```diff
@@ -166,6 +225,15 @@ void testTlsExternalUserWithQuotas() {
         testUserWithQuotas(tlsExternalUser);
     }
 
+    @TestDoc(
```

Comment:
If you have documentation for the tests encapsulating the original one, should the `testUserWithQuotas` have documentation as well?

Reply:
> You can use TestDoc here as well or?

True, I didn't know about it. I thought it's conditional by `@Test` annotation or smth that. Anyway, I have added doc also for quotas test case. Thanks! :)